README.md

```diff
@@ -135,12 +137,10 @@ The table below summarizes the results of running various ML models through our
 | aten.unsqueeze.default         | ✅       |       1 |
 | aten.view.default              | ✅       |     283 |
 #### Llama
-| aten ops              | status   |   count |
-|:----------------------|:---------|--------:|
-| aten._to_copy.default | ✘        |       1 |
-| aten.mm.default       | ✅       |       1 |
-| aten.t.default        | ✅       |       1 |
-| aten.view.default     | ✅       |       2 |
+| aten ops               | status   |   count |
+|:-----------------------|:---------|--------:|
+| aten.slice.Tensor      | ✘        |       1 |
+| aten.unsqueeze.default | ✘        |       2 |
```

Comment:
llama conversion regressed

Reply:
I guess this is because of multiple subgraphs in Llama model. This one is capturing a different subgraph compared to the last one, hence different results. @kevinwuTT We need to take into account multiple subgraphs due to dynamic control flow statements in models.